src/parameterizations/vertical/MOM_kappa_shear.F90

```diff
@@ -1873,6 +1882,10 @@ function kappa_shear_init(Time, G, GV, US, param_file, diag, CS)
                  "If true, do the calculations of the shear-driven mixing "//&
                  "at the cell vertices (i.e., the vorticity points).", &
                  default=.false., do_not_log=just_read)
+  call get_param(param_file, mdl, "VERTEX_SHEAR_VISCOSITY_BUG", CS%VS_viscosity_bug, &
+                 "If true, use a bug in vertex shear that zeros out viscosities at "//&
+                 "vertices on coastlines.", &
+                 default=.true., do_not_log=just_read)
```

Comment:
Actually, I have one minor change to request here to limit the changes to MOM_parameter_doc files to cases whree this bug-fix flag will matter.  Please change line 1888 to `default=.true., do_not_log=just_read.or.(.not.CS%KS_at_vertex))`